action.yml

```
@@ -3,7 +3,7 @@ description: 'Publish the release'
 
 inputs:
   npm-tag:
-    description: 'The NPM tag to use when checking for the latest published version of a package. Defaults to `latest`.'
+    description: '(Deprecated) This option is ignored, and will be removed in a future release'
```

This might be a bit more clear as well:
```suggestion
    description: '(Deprecated) This option is no longer used, and will be removed in a future release'
```